base_classes/NXbeam.nxdl.xml

```diff
@@ -170,31 +170,48 @@
             <dim index="1" value="m"/>
         </dimensions>
     </field>
-    <field name="incident_polarization" type="NX_NUMBER" units="NX_ANY"
-        deprecated="use incident_polarization_stokes, see https://github.com/nexusformat/definitions/issues/708">
+    <field name="incident_polarization" type="NX_NUMBER" units="NX_ANY">
         <doc>Polarization vector on entering beamline component</doc>
         <dimensions rank="2">
             <dim index="1" value="nP"/>
             <dim index="2" value="2"/>
         </dimensions>
     </field>
-    <field name="final_polarization" type="NX_NUMBER" units="NX_ANY"
-        deprecated="use final_polarization_stokes, see https://github.com/nexusformat/definitions/issues/708">
+    <field name="final_polarization" type="NX_NUMBER" units="NX_ANY">
         <doc>Polarization vector on leaving beamline component</doc>
         <dimensions rank="2">
             <dim index="1" value="nP"/>
             <dim index="2" value="2"/>
         </dimensions>
     </field>
     <field name="incident_polarization_stokes" type="NX_NUMBER" units="NX_ANY">
-        <doc>Polarization vector on entering beamline component using Stokes notation</doc>
+        <doc>
+             Polarization vector on entering beamline component using Stokes notation
+
+             The Stokes parameters are four components labelled I,Q,U,V or S_0,S_1,S_2,S_3.
+             These are defined with the standard Nexus coordinate frame unless it is
+             overridden by an NXtransformations field pointed to by a depends_on attribute.
+             The last component, describing the circular polarization state, is positive
+             for a right-hand circular state - that is the electric field vector rotates
+             clockwise at the sample and over time when observed from the source.
```

Comment:
Because sign conventions are needed for all polarized comments, I would favor a uniform approach as suggested below.